vscode-extension/editor/src/VSCodeTheme.ts

```diff
@@ -0,0 +1,218 @@
+import { MantineThemeOverride } from "@mantine/core";
+
+// For quicker theme devX, use this theme to pass through to the AIConfigEditor instead of
+// the static VSCodeTheme packaged with the editor. We will consolidate later.
+export const VSCODE_THEME: MantineThemeOverride = {
```

Comment:
This is just a complete copy/paste from existing theme with two changes (commented below), in editorBackground and promptContainer